orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -326,7 +338,21 @@ func (m *swiftDialogMDMMigrator) waitForUnenrollment() error {
 }
 
 func (m *swiftDialogMDMMigrator) renderMigration() error {
-	message, flags, err := m.getMessageAndFlags()
+	log.Debug().Msg("checking manual enrollment status")
+	manualProfileCheck, err := profiles.IsManuallyEnrolledInMDM()
+	if err != nil {
+		return err
+	}
+
+	// Check if we're in a manual migration.
+	migrationType, err := m.mrw.GetMigrationType()
+	if err != nil {
+		log.Error().Err(err).Msg("getting migration type")
```

Comment:
sanity check: if we can't get the migration type we assume automatic, is that right?

Reply:
Once we're handling the ADE case, `GetMigrationType` will return either `manual` or `ade`

Reply:
okay yeah, I meant for the statement below sorry! 

```go
isManual := manualProfileCheck || migrationType == constant.MDMMigrationTypeManual
```

Reply:
it was poorly worded but I meant, we either error here or we default to a migration type. What's the plan? because it currently defaults to ADE